fr/conf.py

```diff
@@ -117,7 +117,7 @@
 # The name of an image file (within the static path) to use as favicon of the
 # docs.  This file should be a Windows icon file (.ico) being 16x16 or 32x32
 # pixels large.
-html_favicon = 'en/_static/favicon.ico'
+html_favicon = '_static/favicon.ico'
```

Comment:
Hello,

No, I've just fix for french folder.



Le 05/08/2016 à 12:15, Serghei Iakovlev a écrit :

In fr/conf.py<https://github.com/phalcon/docs/pull/850#discussion_r73671124>:

> @@ -117,7 +117,7 @@
>  # The name of an image file (within the static path) to use as favicon of the
>  # docs.  This file should be a Windows icon file (.ico) being 16x16 or 32x32
>  # pixels large.
> -html_favicon = 'en/_static/favicon.ico'
> +html_favicon = '_static/favicon.ico'


@SidRoberts<https://github.com/SidRoberts> So, we have to fix this for all languages?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/phalcon/docs/pull/850/files/c363329e2892b65ee650956701977541b5dfb111#r73671124>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AHYK6303qGgaERFiXyzUcg03xqu58fODks5qcw0vgaJpZM4Jdh8t>.